weldx/transformations/cs_manager.py

```diff
@@ -1377,6 +1393,19 @@ def get_cs(
                     lcs.reset_reference_time(time_ref_interp)
                 else:
                     lcs = lcs.interp_time(time_interp, time_ref_interp)
+
+            if invert:
+                if isinstance(lcs.coordinates, TimeSeries):
+                    raise Exception(
+                        "The chosen transformation is time dependent, but no time is "
+                        "given. This is usually the case if the time dependencies are "
+                        "only described by mathematical expressions. Provide the "
+                        "desired time using the corresponding parameter to solve this "
+                        "issue."
+                    )
```

Comment:
I think we can skip the test here since `lcs.invert()` will basically raise the same exception.

Reply:
Just thought that the exception raised by the lcs might be a bit cryptic for a CSM user without deeper knowledge of the underlying mechanisms. Does it hurt here?

Reply:
I would like to keep the code and branches as short as possible, we will have to change `isinstance(lcs.coordinates, TimeSeries):` here again later anyway for example

then see what really is confusing or an issue later
but let's just leave it in here for now